doc/nrf/protocols/multiprotocol/index.rst

```diff
@@ -46,3 +46,59 @@ Given the nature of the dynamic multiprotocol solution, take into account the fo
   The application logic must take this into account by providing a callback for all crucial transmissions and handle the ``MAC_TRANSACTION_OVERFLOW (== 241)`` status code.
 * The most time-critical part of the Zigbee stack operation is the Base Device Behavior (BDB) commissioning process.
   Postpone time-consuming operations until the ``ZB_BDB_SIGNAL_DEVICE_FIRST_START`` or ``ZB_BDB_SIGNAL_DEVICE_REBOOT`` signal is received.
+
+.. _multiprotocol_bt_thread:
+
+Bluetooth LE and Thread coexistence
+***********************************
+
+The Thread protocol has a lower priority than Bluetooth LE.
+When Bluetooth LE consumes a significant amount of radio time, such as during Bluetooth LE central start scanning or to maintain a large number of connections, this can lead to insufficient time left for Thread to access the radio.
+To maintain effective Thread communication, it is essential to reduce the radio time for Bluetooth LE.
+This can be achieved by adjusting the scan and connection parameters.
+
+Bluetooth LE scan parameters
+============================
+
+The key parameters for scanning are:
+
+- *Scan interval* - The time between consecutive scans.
+- *Scan window* - The amount of time within the scan interval that is dedicated to actual scanning.
+
+By reducing the scan window for a given scan interval, the actual radio time used by Bluetooth LE decreases, thereby freeing up more time for Thread.
+Similarly, increasing the scan interval for a set scan window also frees up more radio time.
+Note that this is not purely about the overall percentage of time used by Bluetooth LE.
+
+For example, while both a 20 ms scan window with a 60 ms interval and a 100 ms window with a 300 ms interval consume the same 33% percentage of time, they do not have the same effect.
+The key difference lies in how long Bluetooth LE consistently occupies the radio during active scanning.
+If the scan window is too long, the likelihood of Thread transmissions clashing with Bluetooth LE activity rises, leading to potential collisions and communication delays.
```

Comment:
I would change the end of this sentence a bit, as it's not necessarily about communication delays. Potential collisions lead to transmission errors, that may result in delay (thanks to retransmissions), but when there is too little time left for Thread the communication will not work at all. I think we could just say that collisions lead to transmission errors.

Reply:
We could the sentence with "leading to potential collisions and communication delays, and even transmission errors"?

Reply:
If we get a clash it's always transmission error. The question is whether it's only single transmission error fixed by retransmission or whole communication tear down, because all transmissions are failing for the longer time. So to be true it should be something like "leading to potential collisions and in result communication delays, or even complete communication breakdown"

Reply:
Getting a bit long, so suggesting a split into two sentences:
```suggestion
If the scan window is too long, the likelihood of Thread transmissions clashing with Bluetooth LE activity rises.
This can lead to collisions, and can result in communication delays or even complete communication breakdown.
```